### ISSUE TITLE
Custom icons via Icon Shortcodes plugin

### ISSUE THREAD

**Issue by NomarCub** [Ellpeck/ObsidianCustomFrames#29]
Pane icons could be extended arbitrarily using [this](https://github.com/aidenlx/obsidian-icon-shortcodes) plugin's API.

**Comment by Ellpeck** [comment]
Sorry, but I'm not currently interested in adding compatibility between Custom Frames and other plugins. 

Custom Frames pane icons currently use Obsidian's default icon API, which allows Lucide icons only. For most cases, that should be enough icons, and they fit Obsidian's style pretty nicely.

**Comment by NomarCub** [comment]
This is not a question of "compatibility between Custom Frames other plugins", but of customizing this plugin (with help from another). There's no incompatibility now to solve, I intended this as a request for a new feature. The icon you have for Keep is nice, (and the lightbulb one I found recently is even better for me) but I'd like the pane icons to be more customizable.

That said, I get that you're not interested. I might make a pull request for this in the future.
